test/unit/argument_parser/detail/format_help_test.cpp

```diff
@@ -223,6 +214,71 @@ TEST(help_add_test, add_option)
                            "SeqAn version: 3.0.0");
     EXPECT_TRUE(ranges::equal((stdout   | ranges::view::remove_if(is_space)),
                                expected | ranges::view::remove_if(is_space)));
+}
+
+TEST(test_add_copyright, copyright)
+{
+    std::string stdout;
+    std::string expected;
+
+    // Tests the --copyright call.
+    const char * argvCopyright[] = {"./copyright", "--copyright"};
+    argument_parser copyright("copyright", 2, argvCopyright);
```

Comment:
```suggestion
    argument_parser copyright("myApp", 2, argvCopyright);
```
thats probably why your test fails

Reply:
Whoops, was a weird thing that happened while rebasing. Thanks!